cmake/libprotobuf.cmake

```diff
@@ -116,6 +116,9 @@ if(protobuf_HAVE_LD_VERSION_SCRIPT)
     LINK_DEPENDS ${protobuf_SOURCE_DIR}/src/libprotobuf.map)
 endif()
 target_link_libraries(libprotobuf PRIVATE ${CMAKE_THREAD_LIBS_INIT})
+target_include_directories(libprotobuf
```

Comment:
I think we probably need a similar change in libprotobuf-lite.cmake.

Reply:
Ah ofc, I was testing with protobuf::Message so that didn't get picked up